docs/DeveloperGuide.md

```diff
@@ -269,64 +270,59 @@ shows the updated Sequence diagram for the executing of our `delete` command.
 
-##### Obtaining `Resident` fields
+For more information on possible performance issue, we refer you to the GitHub issue [here](https://github.com/javafxports/openjdk-jfx/issues/409).
 
-From the [documentation](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/TableColumn.html), a
-`TableView` is made up of a number of `TableColumn` instances. `TableColumn` provided us with a method to
-`setCellValueFactory` which allows us to iterate through the list of `Residents` and obtain the value dynamically.
+* Alternative 2: List
 
-In using the `setCellValueFactory` method, we also used the `PropertyValueFactory` class. The implementation of
-`PropertyValueFactory` has enabled us to easily obtain fields due to its *method matching*
-[functionality](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/cell/PropertyValueFactory.html).
+Pros:
+* Ability to customize the layout of the data to be displayed
+* Ability for the user to more focused by viewing lesser amounts of information at a glance
 
-For example, by constructing `nameCol.setCellValueFactory(new PropertyValueFactory<Resident, String>("name")`,
-the "name" string is used as a reference to an assumed `Resident::getName` in `Resident.java`.
-As a consequence of this, for all fields in `Resident`, we have to implement and have implemented suitable methods to
-take advantage of this functionality. The only caveat to this is potentially the complexity of fields that a `Resident`
-could possess. But for our purposes, our fields are mainly represented as `String`, and there are no issues thus far.
+Cons:
+* Possible lack of information displayed at a glance
+* Less intuitive for handling large numbers of entries
 
-<br>
 
-##### Differences in Updating Data
+For the purposes of the user, who has to deal with large amounts of residential information, we opted the use of the table.
+
 
-Another difference between `PersonListPanel` and `ResidentTableView` is the behavior in propagating changes in the
-`Model` component to the `Ui` component. In `ResidentTableView` modifications to any fields of a `Resident` would not require explicit
-invocation of a method to update the Ui. This design was possible as `TableView` automatically adds an observer
-to the returned value from `setCellValueFactory`, as mentioned in the [section above](#obtaining-resident-fields).
-As a result, any updates to `ObservableList<Resident>` would be reflected immediately in all cells of the Table.
+#### Booking Information
+
+Similar to the display of resident information, the `VenueTabView` contains a `BookingTableView` which was also implemented
+via the `TableView` class of `JavaFX`. Here, each row corresponds to the `Day`, and each column corresponds to the `TimePeriod`.
+
+Unlike the `ResidentTableView`, we did not discuss any other alternatives as to how booking information should be displayed as
+we thought that the use of a Table was sufficiently proficient in displaying a timetable.
```

Comment:
Actually if you want, we can add in something here lol. I was thinking of implementing it through a 8x16 grid of JavaFX containers, i.e. `HBox`, `VBox` and so on. So like 8 `HBox`s, which each contain 16 other kinds of containers, for e.g. `VBox` or even another `HBox`.

As for the pros and cons,
The above method I've mentioned would be really time consuming and difficult to implement, but it will look good ah hahaha. (Can do colors and what not)